### PR TITLE
🏑 Fix msg type key for msg export from archives

### PR DESCRIPTION
**Pull request by norkans7**

### CODE REVIEW

File: temba/msgs/models.py [hunk 1]
```diff
@@ -1208,7 +1208,7 @@ def as_archive_json(self):
             "channel": {"uuid": str(self.channel.uuid), "name": self.channel.name} if self.channel else None,
             "urn": self.contact_urn.identity if self.contact_urn else None,
             "direction": Msg.DIRECTIONS.get(self.direction),
-            "msg_type": Msg.MSG_TYPES.get(self.msg_type),
+            "type": Msg.MSG_TYPES.get(self.msg_type),
```
Comment (dodobas) [inline temba/msgs/models.py]: maybe we should add a test for this?

File: .travis.yml [hunk 2]
```diff
@@ -40,6 +42,8 @@ install:
   - npm run build
 
 before_script:
+  - wget -q --waitretry=1 --retry-connrefused -T 10 -O - http://127.0.0.1:9200
```
Comment (rowanseymour) [inline .travis.yml]: this unrelated?

Comment (norkans7) [reply .travis.yml]: Elasticsearch was failing to install on Travis

Comment (norkans7) [reply .travis.yml]: I changed how we install to not require sudo

**Pull request merged**